pkg/arvo/lib/vere.hoon

```diff
@@ -1,6 +1,7 @@
 ::  runtime support code
 ::
-=>  ..zuse  =>
+/+  ethereum, azimuth
+=>  [ethereum=ethereum azimuth=azimuth ..zuse]  =>
```

Comment:
This won't work, because of the way we build `lib/vere` in `+ivory`. I should really add a comment to the top noting that %ford runes aren't currently supported in `lib/vere`.

In the meantime, each of the relevant `|dawn` arms could just be modified to accept a contract address.

Reply:
It looks like it's just be `+czar:give`, `+point:give`, and `+turf:give`, I think it would be fine to just hardcode the contract address in the binary and pass it to those gates.

Reply:
Alternately, and more easily, you could keep just the addresses in %zuse.

Reply:
Right now zuse knows barely anything about azimuth anymore. Seems good to keep that as minimal as possible.

Since we need more than just the contract addresses, I ended up copying the relevant logic over from the libraries, directly into `/lib/vere`. Not the most elegant, but I'm assuming this is just a temporary solution.

Reply:
Oh, of course. This is probably fine for now (it's easy to update later). Or we could just hardcode the dependencies in `gen/ivory`.